misc/version_updater.py

```diff
@@ -414,7 +414,12 @@ def refresh_npm_package_lock(update_files: set[Path]) -> set[Path]:
     },
     ROOT_DIR / "Cargo.toml": {
         Tool.License: [TOML_LICENSE_FIELD],
-        Tool.Parsec: [ReplaceRegex(r'^version = ".*"$', 'version = "{version}"')],
+        Tool.Parsec: [
+            ReplaceRegex(
+                r'^version = ".*" (\s*)# __PARSEC_VERSION__$',
```

Comment:
```suggestion
                r'^version = ".*"\W*# __PARSEC_VERSION__$',
```

- remove parenthesis given we don't need the group in the output
- remove the whitespace between the quote and the parenthesis to reduce the assumptions we rely on (if for whatever reason we end up with `version = "1.2.3"#__PARSEC_VERSION__`, then the code is still valid Rust but our assumption would break this script)
- `\W` to match any whitechar (so also `\t`), again to reduce the amount of assumptions we rely on

Reply:
- `\W` matches any non-word character (equivalent to `[^a-zA-Z0-9_]`)
- `\s` matches any whitespace character (equivalent to `[\r\n\t\f\v ]`)

That almost the same

Reply:
Fixed the multiple space issue in another way